resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -211,6 +276,40 @@ public class ApiClient implements AutoCloseable {
         throw new RuntimeException("No OAuth2 authentication configured!");
     }
 
+    public void setBasePath(String basePath) {
+        this.basePath = basePath;
+    }
+
+    /**
+     * Helper method to apply logging settings to logger 
+     */
+    public void applyLoggingConfiguration() {
```

Comment:
This should be private

Reply:
This is actually being used in the Configuration.java file, so I changed to a default scope.  It's being used when autoReloadConfig is enabled and a config file is deleted from the directory.  In this case, I designed it so that the program reverts to the logging configuration object and applies it with this method. I can remove this behavior and instead take no action on delete if needed?

Reply:
Yeah I think removing that behavior and taking no action on delete is best